cross/android/arm64/tryrun.cmake

```diff
@@ -0,0 +1,128 @@
+# Need to verify this
+SET( HAVE_COMPATIBLE_EXP_EXITCODE 
```

Comment:
I will double check this - @cydhaselton do you still have a termux environment where you can invoke the CoreCLR build scripts on an Android device?